lib/rules/shorthand-property-no-redundant-values/index.mjs

```diff
@@ -134,6 +175,57 @@ const rule = (primary) => {
 
 			if (!isSupportedShorthand(normalizedProp)) return;
 
+			// Special handling for `border-radius` when a slash is present
+			if (normalizedProp === 'border-radius') {
+				const { horizontal, vertical, sawSlash } = splitBorderRadius(value);
+
+				if (sawSlash) {
```

Comment:
[suggestion] To make the main rule function more readable, how about extracting a function like `splitBorderRadius`? E.g.

```js
if (sawSlash) {
	processBorderRadiusValue(result, decl, horizontal, vertical);
	return;
}
```

Reply:
Thank you, I've fixed it.
https://github.com/stylelint/stylelint/pull/8539/commits/c9b5ec8162ccbb08fad9f392684a9ce1ee227b6